src/Schema/Builder/AbstractColumn.php

```diff
@@ -30,16 +30,32 @@
 abstract class AbstractColumn implements ColumnInterface
 {
     /**
-     * Internally used constants representing categories that abstract column types fall under.
-     *
-     * {@see $categoryMap} For mappings of abstract column types to category.
+     * Define the type of the column as a primary key.
      */
     public const CATEGORY_PK = 'pk';
```

Comment:
Hmm... so these are types? Maybe it worth changing the constants name from `CATEGORY_` to `TYPE_`?

Reply:
Are you sure that the phpdoc is correct then? It doesn't mention anything about this and states that it's type.